src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.FileReaderService;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class FileReaderServiceImplTest {
+    private static final String VALID_FILE_TO_READ = "src/main/resources/input.csv";
```

Comment:
If i delete this file test will fail. I believe it is better to create one before test and remove after.